test/io.jl

```diff
@@ -490,4 +490,11 @@ module TestIO
     # Need to wrap macro call inside eval to prevent the error from being
     # thrown prematurely
     @test_throws ArgumentError eval(:(csv"foo,bar"a))
+
+    #Test HTML output for IJulia and similar
+    df = DataFrame(Fish = ["Suzy", "Amir"], Mass = [1.5, Nullable{Float64}()])
```

Comment:
FWIW, you don't actually need to specify `{Float64}` since promotion does that for you.